### ISSUE TITLE
Show samples of the data in Step 3 - relation discovery

### ISSUE THREAD

**Issue by tomas-knap** [odalic/odalic-ui#313]
In Step 3 - graph - it could be interesting to show samples of data for the columns (e.g. in the space right to the graph)

To be considered.

**Comment by brodecva** [comment]
@tomas-knap This is pretty bold proposal, considering the time frame left.

**Comment by tomas-knap** [comment]
Yes, sure. That is not planned to be done till end of Feb. If there is no milestone assigned, than that is future work
